gokart/utils.py

```diff
@@ -71,6 +71,19 @@ def flatten(targets: FlattenableItems[T]) -> list[T]:
     return flat
 
 
+K = TypeVar('K')
+
+
+def map_flattenable_items(items: FlattenableItems[T], func: Callable[[T], K]) -> FlattenableItems[K]:
```

Comment:
https://docs.python.org/3.13/library/functions.html#map

python original map define `map(function, iterable)`, so you must suit python's manner. 

```suggestion
def map_flattenable_items(func: Callable[[T], K], items: FlattenableItems[T]) -> FlattenableItems[K]:
```